src/pages/library/datasetBuilder/CohortEditor.test.ts

```diff
@@ -178,8 +178,12 @@ describe('CohortEditor', () => {
       updateCriteria,
     });
     // Act
-    screen.getByLabelText(`${criteria.name} low slider`);
-    screen.getByLabelText(`${criteria.name} high slider`);
+    fireEvent.keyDown(screen.getByLabelText(`${criteria.name} low slider`), { keyCode: 39 /* Right Arrow */ });
+    fireEvent.keyDown(screen.getByLabelText(`${criteria.name} high slider`), { keyCode: 37 /* Left Arrow */ });
```

Comment:
Suggest adding a comment explaining why this uses `fireEvent` vs `userEvent` to save time for anyone who attempts to convert this to `userEvent` in the future.